container/build_tar.py

```diff
@@ -259,7 +272,16 @@ def add_deb(self, deb):
       while current:
         parts = current.filename.split(".")
         name = parts[0]
-        ext = '.'.join(parts[1:])
+        if parts[-1] == 'xz':
```

Comment:
Could you try to do the import of lzma only if needed (i.e., inside this conditional, as in this PR: https://github.com/bazelbuild/rules_docker/pull/433/files)